app/experimenter/visualization/api/v3/views.py

```diff
@@ -216,7 +218,15 @@ def generate_results_object(data, experiment, window="overall"):
 
 def get_data(slug, window):
     filename = f"""statistics_{slug}_{window}.json"""
-    data = load_data_from_gcs(filename)
+    path = os.path.join(STATISTICS_FOLDER, filename)
+    data = load_data_from_gcs(path)
+    return data
+
+
+def get_metadata(slug):
+    filename = f"""metadata_{slug}.json"""
```

Comment:
Does this need to be triple quoted?

Reply:
Oh also noticed the other one above is too

Reply:
no it doesn't lol good point